src/commonTest/kotlin/suite/TestSuitePlaces.kt

```diff
@@ -0,0 +1,133 @@
+package suite
+
+import clientPlaces
+import com.algolia.search.client.ClientPlaces
+import com.algolia.search.model.ObjectID
+import com.algolia.search.model.places.Country
+import com.algolia.search.model.places.PlaceType
+import com.algolia.search.model.places.PlacesQuery
+import com.algolia.search.model.search.AroundRadius
+import com.algolia.search.model.search.Point
+import com.algolia.search.model.search.Language
+import runBlocking
+import shouldEqual
+import shouldNotBeEmpty
+import kotlin.test.Test
+
+
+internal class TestSuitePlaces {
+
+    @Test
+    fun anonymous() {
+        runBlocking {
+            ClientPlaces().searchPlaces()
```

Comment:
Nothing to test on the result? :thinking: 

We could at least:
```suggestion
            ClientPlaces().searchPlaces().hits.shouldNotBeEmpty()
```

Reply:
Yes